vscode/src/azure/networkRequests.ts

```diff
@@ -255,98 +266,6 @@ export class StorageUris {
   }
 }
 
-export async function checkCorsConfig(token: string, quantumUris: QuantumUris) {
```

Comment:
Again, all dead code now we use the storage proxy in the service. It's been unused for a while.